packages/compass-e2e-tests/helpers/commands/get-connect-form-connection-string.ts

```diff
@@ -0,0 +1,18 @@
+import type { CompassBrowser } from '../compass-browser';
+import * as Selectors from '../selectors';
+
+export async function getConnectFormConnectionString(
+  browser: CompassBrowser,
+  isFocused = false
```

Comment:
I’d maybe consider renaming this parameter because `isFocused` sounds like it describes the current state at the time of the call, not what this function actually does (which is to *make* it focused if the argument is `true`)

```suggestion
  shouldFocusInput = false
```